### PR TITLE
update bash completions for push and pull

### PR DESCRIPTION
**Pull request by jessfraz**
ping @albers @tianon this was driving me nuts ;)

### CODE REVIEW

**Comment by tianon** [comment]
Wait, people actually run with this enabled? :innocent:

(jk jk) LGTM

**Comment by jfrazelle** [comment]
BAHAHAHAHA

File: contrib/completion/bash/docker [hunk 1]
```diff
@@ -1466,7 +1466,7 @@ _docker_ps() {
 _docker_pull() {
 	case "$cur" in
 		-*)
-			COMPREPLY=( $( compgen -W "--all-tags -a --help" -- "$cur" ) )
+			COMPREPLY=( $( compgen -W "--all-tags -a --disable-content-trust --help" -- "$cur" ) )
```
Comment (albers) [inline contrib/completion/bash/docker]: As this option defaults to `true`, you could as well complete `--disable-content-trust=false` as in the completions for `build` and `run`.

Comment (jfrazelle) [inline contrib/completion/bash/docker]: oooooo very nice

On Fri, Jan 29, 2016 at 10:12 AM, Harald Albers <notifications@github.com>
wrote:

> In contrib/completion/bash/docker
> <https://github.com/docker/docker/pull/19836#discussion_r51293911>:
>
> > @@ -1466,7 +1466,7 @@ _docker_ps() {
> >  _docker_pull() {
> >  	case "$cur" in
> >  		-*)
> > -			COMPREPLY=( $( compgen -W "--all-tags -a --help" -- "$cur" ) )
> > +			COMPREPLY=( $( compgen -W "--all-tags -a --disable-content-trust --help" -- "$cur" ) )
>
> As this option defaults to true, you could as well complete
> --disable-content-trust=false as in the completions for build and run.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/19836/files#r51293911>.
>



-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>

**Comment by albers** [comment]
These inverse options are driving me nuts. And `--disable-content-trust=false` is really obscure! You have to disable disabling?

**Comment by albers** [comment]
LGTM, thanks Jess!

**Comment by jfrazelle** [comment]
lol right? its the worst one

On Fri, Jan 29, 2016 at 10:26 AM, Harald Albers <notifications@github.com>
wrote:

> These inverse options are driving me nuts. And
> --disable-content-trust=false is really obscure! You have to disable
> disabling?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/19836#issuecomment-176896321>.
>



-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>

**Comment by calavera** [comment]
LGTM

**Comment by thaJeztah** [comment]
@albers I think the reason for the double negative here is because there were/are plans to make it the default in a future version; after that, it'll just become `--disable-content-trust` to disable

**Pull request merged**